### PR TITLE
Match progress info

### PR DESCRIPTION
**Pull request by Fueredoriku**
A progress display for each player on scoreboards after every round

### CODE REVIEW

File: Assets/Scripts/Scoreboard/Scoreboard.cs [hunk 1]
```diff
@@ -51,11 +60,14 @@ void Start()
...
     {
-        background.color = player.playerIdentity.color;
+        wantedPoster.GetComponent<Image>().color = player.playerIdentity.color;
         this.subtitle.text = subtitle;
         this.player = player;
+        progressPoster.GetComponent<Image>().color = player.playerIdentity.color;
+        progressDescription.text = player.playerIdentity.playerName;
 
         scoreboardManager.ShowNextCrime += NextStep;
+        scoreboardManager.DisplayProgress += ShowProgress;
```
Comment (toberge) [inline Assets/Scripts/Scoreboard/Scoreboard.cs:70]: The difference between Display and Show is... what, exactly? Some of these names could use a do-over.

File: Assets/Scripts/Scoreboard/ScoreboardManager.cs [hunk 2]
```diff
@@ -140,8 +141,9 @@ private IEnumerator NextCrime()
         }
         else
         {
+            DisplayProgress?.Invoke();
             // Start next round
-            yield return new WaitForSeconds(newCrimeDelay);
+            yield return new WaitForSeconds(5f);
```
Comment (toberge) [inline Assets/Scripts/Scoreboard/ScoreboardManager.cs:146]: This has to match up with the timing in the Scoreboard class, right? Seems easily breakable but will work fine for the time being. Perhaps you could expose the time values in the scoreboards and use them here?

**Review by toberge — APPROVED** [review]
Works great, nice to finally see the match progress :tada:
 (though I stumbled upon some unrelated bugs while testing)

File: Assets/Scripts/Scoreboard/Scoreboard.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/Scoreboard/Scoreboard.cs]: Has been updated 👍

File: Assets/Scripts/Scoreboard/ScoreboardManager.cs (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Scripts/Scoreboard/ScoreboardManager.cs]: I have now exposed the value and made individual scoreboards adjust their timings based on this value 👍

**Pull request merged**